package.json

```diff
@@ -95,7 +95,7 @@
     "react-native-image-zoom-viewer": "^2.2.25",
     "seamless-immutable": "^7.1.4",
     "stream-chat": "^1.14.0",
-    "uuid": "^3.3.2"
+    "uuid": "^8.3.0"
```

Comment:
@nhannah, @vishalnarkhede  This update crash Expo SDK 38, `uuid` require a polyfill for `crypto.getRandomValues `
(https://github.com/uuidjs/uuid#react-native)
But, `import 'react-native-get-random-values';` only support from SDK-38. So, This update crash SendMessage method.

Reply:
> @nhannah, @vishalnarkhede This update crash Expo SDK 38, `uuid` require a polyfill for `crypto.getRandomValues `
> (https://github.com/uuidjs/uuid#react-native)
> But, `import 'react-native-get-random-values';` only support from SDK-39. So, This update crash SendMessage method.

We have dropped it back to v3.x for an upcoming release